src/main/scala/com.snowplowanalytics.iglu/ctl/commands/TableCheck.scala

```diff
@@ -147,7 +174,7 @@ object TableCheck {
   def checkTable(storage: Storage[IO], schemaKeyOfLastVersion: SchemaKey, modelGroupSchemas: List[IgluSchema], dbSchema: String): Failing[TableCheckResult] = {
     val res = for {
       existingColumns <- storage.getColumns(SchemaDDLStringUtils.getTableName(SchemaMap(schemaKeyOfLastVersion)), dbSchema)
-      expectedColumns <- IO.pure(buildTableDdl(modelGroupSchemas))
+      expectedColumns = buildTableDdl(modelGroupSchemas)
     } yield expectedColumns.map(checkColumns(schemaKeyOfLastVersion, existingColumns, _))
```

Comment:
Not sure why do we still need `.map` here if its not `IO`?

Reply:
It is `Either` and I am trying to reach the right of it in there.

Reply:
Not following. Why it was `IO[Either]` before then?

Reply:
Making IO was unnecessary actually, I realized after your comment.

Reply:
I'm just not sure how it compiled then.

Reply:
It was like that before 
```
storage.getColumns().flatMap { existingColumns => 
  IO.pure(buildTableDdl(modelGroupSchemas)).map { expectedColumns: Either =>
    expectedColumns.map(checkColumns(schemaKeyOfLastVersion, existingColumns, _))
  }
}
```
Now it is like this according to my understanding:
```
storage.getColumns().map { existingColumns => 
  val expectedColumns: Either = buildTableDdl(modelGroupSchemas)
  expectedColumns.map(checkColumns(schemaKeyOfLastVersion, existingColumns, _))
}
```